api/api.ipr

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
OK, makes sense.

Could you add a comment at the top saying something like that? (Or would that get overwritten the next time something updates the file?)